lochness/xnat/__init__.py

```diff
@@ -173,13 +176,24 @@ def download_xnat_session_dataorc(
         subject (str): The ID of the subject.
         session (str): The ID of the session to download.
         out_dir (str): The directory to which the session data should be downloaded.
+        xnat_username (str): The username to use when authenticating with the XNAT server.
+        xnat_password (str): The password to use when authenticating with the XNAT server.
 
     Returns:
         None
     """
 
+    dataorc_binary_path = Path('/opt/software/dataorc-bin/dataorc')
```

Comment:
Can we make it generic here? Here is my idea:

* keep only dataorc here and add `/opt/software/dataorc-bin/` to PATH before launching lochness
* you could also do a sanity check before this line to confim dataorc is present in PATH. That might be an overkill though.

Reply:
With the latest commit, I now find `dataorc` path from PATH environment variable and use that. I also raise an exception if `dataorc` is not in PATH.